### DISCUSSION TITLE
feature request: show versions of non @version formulae when listed

### DISCUSSION THREAD

**Discussion by rdp** [Homebrew/discussions#489] [Everyday usage]
It would be nice if the following output told me what version "openjdk" is:

```
$ brew search openjdk
==> Formulae
openjdk ✔                                                                 openjdk@11 ✔                                                              openjdk@8
```

Suggestion/feature request: show "openjdk [v15.1]" or something like that.  Let me know if this isn't the right forum for such requests.  Thanks all.

**Comment by rdp** [comment]
Another instantiation.  `brew search`  lists the old versions but not the version of the current...FWIW :)

```
% brew search postgresql
==> Formulae
postgresql ✔        postgresql@10       postgresql@11       postgresql@12       postgresql@13       postgresql@9.4      postgresql@9.5      postgresql@9.6      qt-postgresql      ...
```

**Comment by SMillerDev** [comment]
Feature requests should be made as pull requests.
